docs/source/fermi-integrals.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>nitpicking, but its missing <code>dx</code> in the integral. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pynucastro/pynucastro/pull/1046/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
fixed